developer/env/armada.env

```diff
@@ -0,0 +1 @@
+ARMADA_SCHEDULING_PREEMPTION_DEFAULTPRIORITYCLASS=""
```

Comment:
Created seperate env files to stop future potential conflicts.

i.e `ARMADA_POSTGRES_CONNECTION_DBNAME` for lookout or lookoutv2? there is no distinction.